configuration/config2/EnvironmentEventHandlers.cpp

```diff
@@ -1,6 +1,6 @@
 /*##############################################################################
 
-HPCC SYSTEMS software Copyright (C) 2018 HPCC Systems�.
+HPCC SYSTEMS software Copyright (C) 2018 HPCC Systems�.
```

Comment:
this cr symbol looks odd

Reply:
Fixed. I believe the issue is when I create a file in Windows, and push it. I think the character is different between Linux and Windows.